sources/packages/web/src/views/aest/student/SINManagement.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+  <v-card class="mt-4">
+    <div class="mx-5 py-4">
+      <body-header
+        title="Social Insurance Number"
+        subTitle="The first row will always be the student's current active SIN."
+        :recordsCount="studentSINValidations?.length"
+        class="m-1"
+      >
+        <template #actions>
+          <v-btn color="primary" data-cy="addNewSINButton" @click="addNewSIN"
+            ><font-awesome-icon
+              :icon="['fas', 'plus-circle']"
+              class="mr-2"
+            />Add new SIN</v-btn
+          >
+        </template>
+      </body-header>
+      <content-group>
+        <toggle-content :toggled="!studentSINValidations?.length">
```

Comment:
Doubt which is not a part of the PR. is playful-cat the toggle content across the application?

Reply:
As the last check with @hellolynn-tbtb, yes it is. The component has the `playful-cat` defined as default but it is also available to be changed using a v-slot.